extensions/wikia/FliteTag/FliteTagController.class.php

```diff
@@ -0,0 +1,91 @@
+<?php
+class FliteTagController extends WikiaParserTagController {
+	const MIN_SIZE = 1;
+	const PARSER_TAG_NAME = 'flite';
+	const GUID_PATTERN = '/[a-z0-9]{8,8}\-[a-z0-9]{4,4}\-[a-z0-9]{4,4}\-[a-z0-9]{4,4}\-[a-z0-9]{12,12}/';
+
+	private $requiredParams = [ 'guid', 'width', 'height' ];
+
+	public static function onParserFirstCallInit( Parser $parser ) {
+		$parser->setHook( self::PARSER_TAG_NAME, [ new static(), 'renderFliteAdUnit' ] );
```

Comment:
Are you sure it should be `[ new static(), 'renderFliteAdUnit' ]` instead of `'FliteTagController::renderFliteAdUnit' ? Also `renderFliteAdUnit` should be static.